src/mainwindow.cpp

```diff
@@ -436,8 +439,18 @@ void PointViewerMainWindow::handleMessage(QByteArray message)
 
 void PointViewerMainWindow::keyReleaseEvent(QKeyEvent* event)
 {
-    if(event->key() == Qt::Key_Escape)
-        close();
+    switch(event->key())
+    {
+        case Qt::Key_Escape:
+            close();
+            break;
+        case Qt::Key_F11:
```

Comment:
I think this might be better as a global keyboard shortcut, as set using a `QAction` in the view menu?  That way it's a lot more discoverable, since the keyboard shortcut will show up on the menu.

I just noticed in the docs that there's also `setWindowState(Qt::WindowFullScreen)` - if you want a truer full screen that might make more sense too?